packages/rc-tooltip/src/lib/__snapshots__/rc-tooltip.spec.tsx.snap

```diff
@@ -0,0 +1,25 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
```

Comment:
I'm still puzzled why the snapshots look like this. But I guess there is not much to do?